modules/openapi-generator/src/test/resources/2_0/petstore.yaml

```diff
@@ -112,6 +112,37 @@ paths:
         - petstore_auth:
             - 'write:pets'
             - 'read:pets'
+  /pet/findByName:
+    get:
+      tags:
+        - pet
+      summary: Finds Pets by name
+      description: ''
+      operationId: findPetsByName
+      produces:
+        - application/xml
+        - application/json
+      parameters:
+        - name: name
+          in: query
+          description: Name to be used as a filter
+          required: true
+          type: string
+          default: Rover
+          collectionFormat: csv
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Pet'
+        '400':
+          description: Invalid name value
+      security:
+        - petstore_auth:
+            - 'write:pets'
+            - 'read:pets'
   /pet/findByTags:
```

Comment:
Please undo changes to petstore.yaml. We'll create a separate test spec for Kotlin Spring generator and will include this particular test.

Reply:
Removed the petstore.yaml change and its related sample changes.